spec/functional/resource/registry_spec.rb

```diff
@@ -154,8 +188,13 @@ def create_deletable_keys
     end
 
     it "does not create the key if it already exists with same value and type but datatype of data differs" do
+      first_resource = Chef::Resource::RegistryKey.new(resource_name, @run_context)
+      first_resource.key(reg_child)
+      first_resource.values([{ name: "number", type: :dword, data: 12344 }])
+      first_resource.run_action(:create)
+
       @new_resource.key(reg_child)
-      @new_resource.values([{ name: "number", type: :dword, data: "12345" }])
```

Comment:
Not really sure how this registry key was supposed to begin with any value of `"12345"` but I invite anyone else with a sharper eye or knowledge of the codebase to point out what I'm missing. It seemed to be broken because of this.

Reply:
shouldn't i just be `1234`, based on all the stuff below?